### PR TITLE
Add a CONTRIBUTING file to clarify contribution process

### PR DESCRIPTION
**Pull request by ann0see**
As proposed in https://github.com/corrados/jamulus/issues/596#issuecomment-695074557

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+You want to contribute? We really appreciate that!
+Please have a look at this list to faciliate your contribution:
```
Comment (trebmuh) [inline CONTRIBUTING.md:2]: typo: faciliate -> fcilitate

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+* Before you start coding create a Github issue at https://github.com/corrados/jamulus/issues . In that new issue write your specification and that you are planning to do the coding. Then we can discuss the specification. Only start coding after we agreed to a specification.
```
Comment (trebmuh) [inline CONTRIBUTING.md:4]: coding create -> coding, create

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+* If the Github issue for your feature/bug fix already exists, then write a message in that issue that you want to work on it.
+* Respect the existing code style: Tab size=4, insert spaces.
+* Make sure (if possible) that your code compiles on Windows/Mac/Linux.
+* Do not use diff/patch to send your code changes but create a Github fork of the Jamulus code and create a Pull Request when you are done.
+
+Also look here for other guidelines: https://github.com/corrados/jamulus/issues/596
+
+We're looking forward to seing your changes!
```
Comment (trebmuh) [inline CONTRIBUTING.md:12]: typo: seing -> seeing

File: CONTRIBUTING.md (hunk 1 above)
Comment (ann0see) [reply CONTRIBUTING.md:2]: You mean facilitate ;-). Will be fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (ann0see) [reply CONTRIBUTING.md:4]: Thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (trebmuh) [reply CONTRIBUTING.md]: yes, that's what I was trying to mean :)

**Comment by corrados** [comment]
As you further reduced the README size, I will now put the CONTRIBUTING content in that README file.

**Comment by ann0see** [comment]
Better link it. A contributing file is automatically shown on each PR.

**Comment by corrados** [comment]
Oh, I did not know that. Hmm, too late. I already commited my change. You were 10 seconds too late ;-).
I'll fix this.

**Comment by ann0see** [comment]
https://github.blog/2012-09-17-contributing-guidelines/

No problem

**Comment by corrados** [comment]
Ok, it should be fixed now.

**Pull request merged**